packages/mds-db/geographies.ts

```diff
@@ -158,27 +158,34 @@ export async function publishGeography(params: PublishGeographiesParams) {
     conditions.push(`publish_date = ${vals.add(publish_date)}`)
     const sql = `UPDATE ${schema.TABLE.geographies} SET ${conditions} where geography_id=${vals.add(geography_id)}`
 
-    await client.query(sql, vals.values())
-    return geography_id
+    const res = await client.query(sql, vals.values())
```

Comment:
As discussed, one possible solution for this type mismatch that we explored was annotating each function call in `mds-db` with an explicit return type, and requiring that in an `.eslint.js` file for just the `mds-db` package. I discovered that something like a third of `mds-db` doesn't have explicit returns, and it seems to me that it would be out of the scope of this PR to add them in. I'm sticking with just adding explicit returns to `mds-db/geographies.ts`.